ansible/roles/operator-pipeline/templates/openshift/tasks/show-support-link.yml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: show-support-link
+spec:
+  params:
+    - name: env
+      description: Enviroment. One of [dev, qa, stage, production]
+    - name: type
+      description: Connect case type.
+      default: "CERT"
+    - name: source
+      description: Connect case source.
+      default: "tekton"
+    - name: cert_project_id
+      description: Pyxis certification project id.
+    - name: cert_project_type
+      description: Connect cert project type. Should be Operator Bundle Image for operator pipelines.
+      default: "Operator Bundle Image"
+  steps:
+    - name: show-support-link
+      image: quay.io/redhat-isv/operator-pipelines-images
+      script: |
+        #! /usr/bin/env bash
+        set -e
+
+        LINK_ENV="$(params.env)"
+        # url-encode parameters
+        LINK_TYPE=$(printf %s "$(params.type)" | jq -sRr @uri)
+        LINK_SOURCE=$(printf %s "$(params.source)" | jq -sRr @uri)
+        CERT_PROJECT_ID=$(printf %s "$(params.cert_project_id)" | jq -sRr @uri)
+        CERT_PROJECT_TYPE=$(printf %s "$(params.cert_project_type)" | jq -sRr @uri)
+
+        # select the correct connect instance
+        URL_ENV="connect"
+        if [[ $LINK_ENV != "production" ]]; then
+            URL_ENV="$URL_ENV.$LINK_ENV"
+        fi
+
+        echo "Follow this link if you wish to open a support case:"
+        echo "https://$URL_ENV.redhat.com/support/technology-partner/#/case/new?type=$LINK_TYPE&source=$LINK_SOURCE&cert_project_id=$CERT_PROJECT_ID&cert_project_type=$CERT_PROJECT_TYPE"
```

Comment:
Please double check with bcook and abhiskum to see if we should always post a prod support link.